build.gradle

```diff
@@ -262,6 +262,36 @@ project(':iceberg-data') {
   }
 }
 
+project(':iceberg-aliyun') {
+  dependencies {
+    compile project(':iceberg-api')
+    compile project(':iceberg-common')
+
+    compileOnly 'com.aliyun.oss:aliyun-sdk-oss'
+    compileOnly 'javax.xml.bind:jaxb-api'
+    compileOnly 'javax.activation:activation'
+    compileOnly 'org.glassfish.jaxb:jaxb-runtime'
```

Comment:
Why do we need to introduce those three jars when adding a testing framework under the aliyun/src/test module?

Reply:
per aliyun oss sdk documentation, https://help.aliyun.com/document_detail/32009.html
jaxb related dependencies need to be added under java 9 and plus version environment.

Reply:
Okay, then I will recommend to package all the dependencies jar into a single bundled iceberg-aliyun jar. Because in flink SQL client,  we will need to add [each jar](https://iceberg.apache.org/flink/#preparation-when-using-flink-sql-client) in the shell as the following: 

```shell
./bin/sql-client.sh embedded \
-j <iceberg-aliyu>.jar \
-j <aliyun-oss>.jar \
-j <javax.xml.bind:jaxb-api>.jar \
-j <org.glassfish.jaxb:jaxb-runtime>.jar 
shell
```

It will be quite tedious for people to add jars one by one to make the iceberg job works ( aws-sdk don't need to package all of them into a bundled jar because aws sdk has provided [it](https://mvnrepository.com/artifact/software.amazon.awssdk/bundle) ( see [document](https://iceberg.apache.org/aws/#enabling-aws-integration)).